editor/editor_export.cpp

```diff
@@ -974,7 +974,11 @@ Error EditorExportPlatform::save_pack(const Ref<EditorExportPreset> &p_preset, c
 	f->store_32(1); //pack version
 	f->store_32(VERSION_MAJOR);
 	f->store_32(VERSION_MINOR);
+#ifdef VERSION_PATCH
+	f->store_32(VERSION_PATCH);
+#else
 	f->store_32(0); //hmph
```

Comment:
Maybe we could have a more meaningful comment here :slightly_smiling_face: 
```suggestion
	f->store_32(0); // Patch version is always 0 for initial minor release versions (e.g. 3.2 is considered to be 3.2.0).
```